.idea/codeStyles/Project.xml

```diff
@@ -12,6 +12,9 @@
       <option name="IF_RPAREN_ON_NEW_LINE" value="false" />
       <option name="CODE_STYLE_DEFAULTS" value="KOTLIN_OFFICIAL" />
     </JetCodeStyleSettings>
+    <ScalaCodeStyleSettings>
```

Comment:
I don't think you intended to add the _scala code style_ to a kotlin project.

Reply:
Reverted.